Source/MLX/Documentation.docc/Organization/shapes.md

```diff
@@ -63,4 +63,6 @@ These methods manipulate the shape and contents of the array:
 - ``split(_:parts:axis:stream:)``
 - ``stacked(_:axis:stream:)``
 - ``swappedAxes(_:_:_:stream:)``
+- ``tiled(_:repetitions:stream:)-72ntc``
+- ``tiled(_:repetitions:stream:)-eouf``
```

Comment:
What is this?

Reply:
Is it the equivalent of `mx.repeat`? If so, what motivated the name change?

Reply:
No, this is different than repeat (which maybe should be `repeated`):

```
      R"pbdoc(
      tile(a: array, reps: Union[int, List[int]], /, *, stream: Union[None, Stream, Device] = None) -> array
```

vs

```
      R"pbdoc(
      repeat(array: array, repeats: int, axis: Optional[int] = None, *, stream: Union[None, Stream, Device] = None) -> array
```

They have different backing (on the C++ side).  The documentation looks pretty similar -- should we have both of these?

Reply:
No we have them sorry I forgot we added them!